scripts/quarantine_windows.yaml

```diff
@@ -0,0 +1,36 @@
+# The configurations resulting as a product of scenarios and platforms
+# will be skipped if quarantine is used. More details here:
+# https://docs.zephyrproject.org/latest/guides/test/twister.html#quarantine
+# To have an empty list use:
+# - scenarios:
+#    - None
+#  platforms:
+#    - None
+
+- scenarios:
+    - asset_tracker_v2.gnss_module_test.tester
+    - asset_tracker_v2.ui_module_test.tester
+    - applications.asset_tracker_v2.cloud.cloud_codec.json_common.aws
+    - applications.asset_tracker_v2.cloud.cloud_codec.json_common.azure
+    - asset_tracker_v2.debug_module_test.tester
+    - samples.edge_impulse.wrapper
+    - samples.event_manager
+  platforms:
+    - native_posix
+    - qemu_cortex_m3
+    - qemu_x86
+  comment: "Cannot build samples for Native POSIX and QEMU on Windows OS"
+
+- scenarios:
+    - asset_tracker_v2.debug_module_test.tester
```

Comment:
Do you need to repeat these three tests here?

Reply:
we agreed that these tests will be only be built  in Linux CI and executed,

Reply:
@balaji-nordic 

Yes, I need to specify those test in separate entry to point out that those tests are in quarantine for all platforms (not only for `native_posix` or `qemu`). 

But you are right that it can be simplify and I decided to remove those tests from first entry ("Cannot build samples for Native POSIX and QEMU on Windows OS"). They will be filtered anyway in second entry ("ruby package not available in Windows toolchain").

Thank you for your comment :)